File: back/src/models/campus.rs
use sqlx::types::chrono::{DateTime, Utc};

#[derive(Debug)]
pub struct Campus {
    pub campus_id : i32,
    pub name : String,
    pub university_id : i32,
    pub state_id : i32,
    pub created_at : DateTime<Utc>,
    pub modified_at : Option<DateTime<Utc>>,
}

File: back-csharp/_data/Campus.cs
using System;
using System.Collections.Generic;

namespace back_csharp._data
{
    public partial class Campus
    {
        public Campus()
        {
            Rosters = new HashSet<Roster>();
        }

        public int CampusId { get; set; }
        public string Name { get; set; } = null!;
        public int UniversityId { get; set; }
        public int StateId { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime? ModifiedAt { get; set; }

        public virtual State State { get; set; } = null!;
        public virtual University University { get; set; } = null!;
        public virtual ICollection<Roster> Rosters { get; set; }
    }
}
